src/ApiCore/OperationResponse.php

```diff
@@ -48,26 +52,41 @@
  */
 class OperationResponse
 {
-    const DEFAULT_POLLING_INTERVAL = 1.0;
+    const DEFAULT_POLLING_INTERVAL = 1000.0;
+    const DEFAULT_POLLING_MULTIPLIER = 2.0;
+    const DEFAULT_MAX_POLLING_INTERVAL = 60000.0;
+    const DEFAULT_MAX_POLLING_DURATION = 0.0;
 
     private $operationName;
     private $operationsClient;
+
     private $operationReturnType;
     private $metadataReturnType;
+    private $defaultPollSettings = [
+        'initialPollDelayMillis' => self::DEFAULT_POLLING_INTERVAL,
+        'pollDelayMultiplier' => self::DEFAULT_POLLING_MULTIPLIER,
+        'maxPollDelayMillis' => self::DEFAULT_MAX_POLLING_INTERVAL,
+        'totalPollTimeoutMillis' => self::DEFAULT_MAX_POLLING_DURATION,
+    ];
+
     private $lastProtoResponse;
     private $deleted = false;
 
     /**
      * OperationResponse constructor.
      *
      * @param string $operationName
-     * @param \Google\ApiCore\LongRunning\OperationsClient $operationsClient
+     * @param OperationsClient $operationsClient
      * @param array $options {
      *                       Optional. Options for configuring the Operation response object.
      *
      *     @type string $operationReturnType The return type of the longrunning operation.
      *     @type string $metadataReturnType The type of the metadata returned in the Operation response.
-     *     @type \google\longrunning\Operation $lastProtoResponse A response already received from the server.
+     *     @type float $initialPollDelayMillis    The initial polling interval to use, in seconds.
```

Comment:
Now that we are using milliseconds, do you think that is enough specificity to use int instead of float?

Reply:
Yep, agreed. Fixed